specifyweb/frontend/js_src/lib/components/QueryBuilder/ResultsWrapper.tsx

```diff
@@ -99,13 +103,16 @@ export function useQueryResultsWrapper({
   recordSetId,
   forceCollection,
   onSortChange: handleSortChange,
+  selectedRows: [selectedRows, setSelectedRows],
+  resultsRef,
 }: ResultsProps): PartialProps | undefined {
   /*
    * Need to store all props in a state so that query field edits do not affect
    * the query results until query is reRun
    */
   const [props, setProps] = React.useState<
-    Omit<PartialProps, 'totalCount'> | undefined
+    | Omit<PartialProps, 'resultsArrayRef' | 'selectedRows' | 'totalCount'>
```

Comment:
`resultsArrayRef` is no longer there. did you forget to rename it to `resultsRef`?

Reply:
yes